client/src/Components/Categories/Categories.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./category.scss";
+
+const Categories = ({ categoryList }) => {
+
+  const orderedCategories = categoryList
+    .filter((item) => item.order != -1)
+    .sort((item1, item2) => {
+      return item1.order - item2.order;
+    });
+    
+  return (
+    <main>
+      {orderedCategories.map((item, key) => {
+        return (
+          <section key={key} className="category-item">
+            <article>
```

Comment:
dont add index to key. Look for any id property in "item"

Reply:
fixed it using id